api-docs/tech-ref-info/check-types/agent-check-type-ref.rst

```diff
@@ -188,8 +188,8 @@ Metrics
 +-----------------+--------------------------------------------------+----------+
 | Metric          | Description                                      | Type     |
 +=================+==================================================+==========+
-| avail           | Available space on the filesystem in kilobytes,  | Int64    |
-|                 | including reserved space.                        |          |
```

Comment:
"Reserved space" would mean the same thing as "used space" correct? Would I need to make this change on the line labeled "free" as well?